scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
```

Comment:
TARS:<br/>2. For the command injection vulnerability:
   Use a library like `shelljs` to execute commands safely, or at least sanitize and validate user input.
```suggestion
import * as shelljs from 'shelljs'

function executeCommand(userInput: string) {
    const sanitizedInput = userInput.replace(/[;&|]/g, '') // Basic sanitization
    shelljs.exec(`ls ${sanitizedInput}`)
}
```